validate-differential-expression/README.md

```diff
@@ -0,0 +1,87 @@
+# refine.bio Examples: Validate Differential Expression
+
+These notebooks compare the differential expression results from a [refine.bio](https://www.refine.bio/) dataset to an independent dataset that has a similar experimental design.
+
+## Background and input requirements
+
+If you have differential expression results from your own data or a collaborator’s data, it is good practice to validate your findings in another dataset that measures a similar tissue with a similar experimental design.
+You can use refine.bio datasets and the steps in this example to quickly check for agreement with your results.
+In order to use this example workflow with your own results, you will need the following fields in your table.
```

Comment:
```suggestion
To use this example workflow with your own results, you will need the following fields in your table:
```

Either moving this below the required fields header and removing the `:` at the end of that header OR deleting this sentence.